core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -98,5 +158,13 @@
     {{ include_js_file('domain/editor/ChangeObjectFactory.js') }}
     {{ include_js_file('domain/editor/UndoRedoService.js') }}
     {{ include_js_file('domain/summaries/ExplorationSummaryBackendApiService.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
+
+    {{ include_js_file('exploration_editor/EditorServices.js') }}
+    {{ include_js_file('exploration_editor/ExplorationEditor.js') }}
+    {{ include_js_file('exploration_editor/GadgetValidationService.js') }}
```

Comment:
``routerServices``, apparently. :( I will be filing another issue to decouple router services from the exploration editor so that it's a cleaner dependency for both editors.